services/project-creator.js

```diff
@@ -18,15 +18,18 @@ function ProjectCreator() {
         authSecret: await new KeyGenerator().generate(),
       };
     } catch (error) {
+      let message;
       if (error.message === 'Unauthorized') {
-        logger.error(`Your session has expired. Please, relogin with the command ${chalk.cyan('lumber run lumber-forestadmin:login')}.`);
+        message = `Your session has expired. Please, relogin with the command ${chalk.cyan('lumber run lumber-forestadmin:login')}.`;
       } else if (error.message === 'Conflict') {
-        logger.error('A project with this name already exists. Please choose another name.');
+        message = 'A project with this name already exists. Please choose another name.';
       } else {
-        logger.error(`An unexpected error occured. Please create a Github issue with following error: ${chalk.red(error)}`);
+        message = `An unexpected error occured. Please create a Github issue with following error: ${chalk.red(error)}`;
```

Comment:
Please reach out for help in our Slack community or create a Github issue with following error:...